packages/block-library/src/navigation-link/index.js

```diff
@@ -2,7 +2,7 @@
  * WordPress dependencies
  */
 import { __, _x } from '@wordpress/i18n';
-import { mapMarker as linkIcon } from '@wordpress/icons';
+import { pageLink as linkIcon } from '@wordpress/icons';
```

Comment:
"Page link" is a bit odd as a term, but don't have a better suggestion. Perhaps "custom link"?

Reply:
Custom link is perfect 👌